packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -224,20 +224,28 @@ export default class GoogleCalendarService implements Calendar {
         version: "v3",
         auth: myGoogleAuth,
       });
+      // Defaulting to primary calendar to avoid error from google api
+      let calendarId = event.destinationCalendar?.externalId ?? "primary";
```

Comment:
We had a conditional but wasn't preventing from sending null to google. We now at least try to delete on primary calendar from the user if it exist.